tests/functional/Updater_Test.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-
 /**
  * @group updates
  */
 class Tribe__Events__Updater_Test extends Tribe__Events__WP_UnitTestCase {
+
 	public function test_constant_updates_applied() {
-		Tribe__Events__Main::instance()->setOption( 'schema-version', 0 );
+		Tribe__Settings_Manager::instance()->set_option( 'schema-version', 0 );
```

Comment:
On `Tribe__Settings_Manager` the method `set_option` is a Static Method. Does it make a diff here?